Kernel_23/doc/Kernel_23/Concepts/FunctionObjectConcepts.h

```diff
@@ -7924,7 +7924,7 @@ class HasOnBoundedSide_3 {
                   const Kernel::Point_3&p); 
 
   /*!
-    return true iff the line segment `ab` is inside the union of the
+    returns true iff the line segment `ab` is inside the union of the
```

Comment:
... but for this file! The file `Kernel_23/doc/Kernel_23/Concepts/FunctionObjectConcepts.h` is full of "returns true" where "true" is not quoted.